src/main/java/com/artipie/rpm/Cli.java

```diff
@@ -102,8 +102,28 @@ public static void main(final String[] args) throws ParseException {
     private static CommandLine parseCommand(final String... args) throws ParseException {
         final CommandLine cmd = Cli.PARSER.parse(Cli.OPTIONS, args);
         if (cmd.getArgs().length != 2) {
-            throw new ParseException("Wrong arguments count, something is missing");
+            throw new Cli.WrongArgumentException();
         }
         return cmd;
     }
+
+    /**
+     * Exception for passing wrong number of arguments.
+     *
+     * @since 0.1
+     */
+    public static final class WrongArgumentException extends ParseException {
```

Comment:
@ammaratef45 We don't need custom exceptions, just check error message explicitly in the test.